commands.js

```diff
@@ -1650,6 +1654,7 @@ var commands = exports.commands = {
 		Rooms.global.send('|refresh|');
 		this.logEntry(user.name + " used /refreshpage");
 	},
+	refreshpagehelp: ["/refreshpage - Forcefully refreshes the page for all users on the server. Requires: ~"],
```

Comment:
Rooms.global.send('|refresh|');

Hm, if it only works for uses in lobby, that wasn't its intent. Maybe we should keep track?